### ISSUE TITLE
Cookie login dosn't work.

### ISSUE THREAD

**Issue by zenz** [r0oth3x49/udemy-dl#407]
Got this error:
```
[*] : Trying to login using cookies ...
[+] : Logged in successfully.
[-] : Udemy Says : 403 Forbidden You should use cookie base method to authenticate or try again in few minutes ...
```
Have to login manually.

**Comment by r0oth3x49** [comment]
@zenz it works fine for me. Did you followed the proper guide in readme.md for extracting cookies? the reason could be your cookies got expired, you need to login and provide the fresh one again cookie session is not for forever it expires within some time.
![Capture](https://user-images.githubusercontent.com/11024397/59961328-9c300c80-94ef-11e9-808b-3432dca73954.PNG)

**Comment by zenz** [comment]
It's really weired. I cannot even purchase a course at that moment on my PC. only after purchased that course with my iOS client, I can watch it on my PC. but udemy-dl with cookie still not working yet.

**Comment by r0oth3x49** [comment]
by the way if it's a udemy account and not SSO you can use the email / password directly to fetch the resources it also works fine.

**Comment by zenz** [comment]
@r0oth3x49 yes, actually I'm using this way currently.

**Comment by swebdevaf** [comment]
same issue here. can't login. and another issue, is downloading incomplete lectures.
